### PR TITLE
Retrieve return value from uploader.request

### PR DESCRIPTION
**Pull request by Nykakin**
If my upload fail, I set 400 error code together with error message. I cannot retrieve this message from code. I want to use:
```python
class SomeUnitTest(unittest.TestCase):
    # ...
    def test_something(self):
        # ...
        try:
            uploader.upload()
        except tusclient.exceptions.TusUploadFailed:
            self.assertEqual(uploader.request.return_value,  'MyErrorMessage')
```

### CODE REVIEW

File: tusclient/request.py [hunk 1]
```diff
@@ -54,6 +61,13 @@ def status_code(self):
         """
         return self.handle.getinfo(pycurl.RESPONSE_CODE)
 
+    @property
+    def return_value(self):
```
Comment (ifedapoolarewaju) [inline tusclient/request.py]: maybe we should rename this property to `response_content`, so it is clearer what it is?

File: tusclient/request.py [hunk 2]
```diff
@@ -20,6 +25,7 @@ class TusRequest(object):
     def __init__(self, uploader):
         self.handle = pycurl.Curl()
         self.response_headers = {}
+        self.storage = io.StringIO()
```
Comment (ifedapoolarewaju) [inline tusclient/request.py]: also should we rename `self.storage` to `self.output` maybe? Or something else that is more explanatory so it's not misleading?

File: tusclient/request.py [hunk 3]
```diff
@@ -54,6 +61,13 @@ def status_code(self):
         """
         return self.handle.getinfo(pycurl.RESPONSE_CODE)
 
+    @property
+    def return_value(self):
+        """
+        Return request ,essage
```
Comment (ifedapoolarewaju) [inline tusclient/request.py]: little typo here :)

**Comment by Nykakin** [comment]
Hey, @ifedapoolarewaju thank you for the feedback! I'm fine with all changes proposed and I've applied them to the code.

**Comment by ifedapoolarewaju** [comment]
@Nykakin thank you for the PR 👍

**Pull request merged**